packages/vc-handler-ld-local/README.md

```diff
@@ -49,3 +49,114 @@ yarn test
 There are other utility scripts that help with development.
 
 - `yarn fix` - runs `eslint --fix` as well as `prettier` to fix code style.
+
+### Request examples:
+
+#### Issue Verifiable Credential
+
+Ed25519:
+
+```shell
+POST http://localhost:7071/api/vdx-ivcs/v1/createVerifiableCredentialLDLocal
```

Comment:
This needs more context. This is using Veramo as a REST API, whilst most will not use it that way. So either make that known, or change the examples to just use the agent directly.

Reply:
Fixed!